### PR TITLE
3D LWFA PICMI Test: Add Electron Beam

### PR DESCRIPTION
**Pull request by EZoni**
Add electron beam to 3D LWFA PICMI tests, following up on #2700. The electron beam is already included in the 2D and RZ examples.

The 1D test was not generating non-zero particle benchmarks, and, since I have not yet understood why, I suggest that we handle the 1D test separately. See #2819 for reference.

### CODE REVIEW

File: Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py [hunk 1]
```diff
@@ -51,6 +51,31 @@
...
+    n_physical_particles = q_tot / q_e,
+    rms_bunch_size = [x_rms, y_rms, z_rms],
+    rms_velocity = [c*ux_th, c*uy_th, c*uz_th],
+    centroid_position = [x_m, y_m, z_m],
+    centroid_velocity = [c*ux_m, c*uy_m, c*uz_m])
+beam = picmi.Species(
+    particle_type = 'electron',
+    name = 'beam',
+    initial_distribution = gaussian_bunch_distribution)
+
```
Comment (EZoni) [inline Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py]: @prkkumar Do you happen to see anything wrong with this beam setup in 1D? I checked the Gaussian beam implementation in 1D and it seems fine to me, however this beam setup seems to produce checksum values for the beam particle attributes that are all zero (see file Regression/Checksum/benchmarks_json/Python_LaserAcceleration_1d.json below). In 2D and 3D the same checksum values are non-zero instead, as expected.

Comment (prkkumar) [reply Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py]: I do not see anything wrong with this setup. Not sure if picmi.GaussianBunchDistribution function needs anything special for 1D though.

Comment (EZoni) [reply Examples/Physics_applications/laser_acceleration/PICMI_inputs_1d.py]: Thanks. I don't see anything specific to 1D (nor any other dimensionality) in `GaussianBunchDistribution` on the PICMI side. I also printed out some of the position and momentum values from within `PhysicalParticleContainer::AddGaussianBeam` (for the beam injection) and they seemed fine to me. I haven't yet understood why the beam checksum values are all zero.

**Review by ax3l — APPROVED** [review]
Thanks! :)

**Pull request merged**